packages/i18n/src/messages/de.json

```
@@ -2167,7 +2223,72 @@
+    "threadSummaryTitle": "Aktuelle Zusammenfassung",
+    "threadSummaryLoading": "Thread-Zusammenfassung wird aktualisiert…",
+    "threadSummaryUpdated": "Aktualisiert {when}",
+    "threadSummaryExpand": "Zusammenfassung erweitern",
+    "threadSummaryCollapse": "Zusammenfassung einklappen",
+    "callCaptureLeaveWarning": "Aufzeichnung ist aus. Zum Bestätigen erneut auf Verlassen klicken.",
+    "callCaptureConsentLocal": "Du zeichnest diesen Anruf auf ({mode}). Andere werden benachrichtigt.",
+    "callCaptureConsentRemote": "{actor} zeichnet diesen Anruf auf ({mode}).",
+    "callCaptureConsentJoin": "Dieser Anruf wird aufgezeichnet ({mode}) von {actor}. Beitritt gilt als Zustimmung.",
+    "callCaptureConsentLegalNote": "Aufzeichnung und Transkript können im Space-Gedächtnis gespeichert werden.",
+    "callCaptureConsentPausedSuffix": " Aufzeichnung ist pausiert.",
+    "callCaptureVoiceStartedRecordingTranscript": "Dieser Anruf wird jetzt aufgezeichnet und transkribiert.",
```

Biome (2.4.15)

[error] The key callCaptureVoiceStartedRecordingTranscript was already declared.

(lint/suspicious/noDuplicateObjectKeys)